parse_solver_output.py

```diff
@@ -119,46 +120,49 @@ def parse_solver_output() -> None:
-                document = ADVISER_STORE.retrieve_document(adviser_id)
-                parameters = document["result"]["parameters"]
-                cli_arguments = document["metadata"]["arguments"]["thoth-adviser"]
-
-                recommendation_type = parameters["recommendation_type"]
-
-                origin = cli_arguments["metadata"]["origin"]
-                github_event_type = cli_arguments["metadata"]["github_event_type"]
-                github_check_run_id = cli_arguments["metadata"]["github_event_type"]
-                github_installation_id = cli_arguments["metadata"]["github_event_type"]
-                github_base_repo_url = cli_arguments["metadata"]["github_event_type"]
-
-                source_type = (cli_arguments.get("metadata") or {}).get("source_type")
-                source_type = source_type.upper() if source_type else None
-
-                # 4. Save adviser_id_message inputs
-                message_input = AdviserReRunContents(
-                    component_name=component_name,
-                    service_version=service_version,
-                    github_event_type=github_event_type,
-                    github_check_run_id=github_check_run_id,
-                    github_installation_id=github_installation_id,
-                    github_base_repo_url=github_base_repo_url,
-                    origin=origin,
-                    recommendation_type=recommendation_type,
-                    re_run_adviser_id=adviser_id,
-                    source_type=source_type,
-                ).dict()
-
-                output_messages.append(
-                    {"topic_name": adviser_rerun_message.base_name, "message_contents": message_input}
-                )
+                # 3. Retrieve adviser inputs to create a new request and schedule adviser with thamos
+
+                retrieved_parameters = True
+                try:
+                    parameters = ADVISER_STORE.retrieve_request(adviser_id)
+                except Exception as retrieve_error:
+                    _LOGGER.error(
+                        f"Failed to retrieve parameters for request with adviser id: {adviser_id}: {retrieve_error}"
+                    )
+                    retrieved_parameters = False
+
+                if retrieved_parameters:
+                    thoth_config = {
+                        "host": "khemenu.thoth-station.ninja",
+                        "tls_verify": False,
+                        "requirements_format": "pipenv",
+                        "runtime_environments": [parameters["runtime_environment"]],
+                    }
+
+                    response = advise_using_config(
+                        pipfile=parameters["application_stack"]["requirements"],
+                        pipfile_lock=parameters["application_stack"]["requirements_lock"],
+                        config=json.dumps(thoth_config),
+                        authenticated=True,
```

Comment:
I don't think this will work, and if it does it shouldn't. Thamos should accept a token as a parameter and have no way of setting "authenticated" since all it does is send the request to the `user-api`